lib/std/Thread.zig

```diff
@@ -713,11 +738,11 @@ const PosixThreadImpl = struct {
                 };
                 return @as(usize, @intCast(count));
             },
-            .solaris, .illumos => {
+            .solaris, .illumos, .serenity => {
                 // The "proper" way to get the cpu count would be to query
                 // /dev/kstat via ioctls, and traverse a linked list for each
                 // cpu.
```

Comment:
Does this comment apply to Serenity? If not, would be good to clarify that it applies to Solarish only.

Reply:
I don't really understand the comment to begin with an why it says proper in quotes, but no it doesn't apply to Serenity - updated :+1:

https://github.com/SerenityOS/serenity/blob/85a53ba32bd705b1fad388093326322423d503b1/Kernel/Syscalls/sysconf.cpp#L20-L22